config/environments/production.rb

```diff
@@ -75,6 +77,9 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
+  # Do not dump schema after migrations.
+  # config.active_record.dump_schema_after_migration = false
```

Comment:
Because I wasn't sure that we wanted this, for sure. I can uncomment it just fine.